composer.json

```diff
@@ -20,21 +20,23 @@
     ],
     "homepage": "https://github.com/Riverline/worker-bundle",
     "require":{
-        "php":                          ">=5.3",
+        "php":                          ">=5.6",
```

Comment:
Why do you change this requirement ?

Reply:
To use constant `::class` but i will revert ...

Reply:
Hum no, bad reason, stomp-php library need php 5.6

Reply:
Ok, no problem ;)